doc/HttpLuaModule.wiki

```diff
@@ -5581,6 +5581,73 @@ According to the current implementation, only the parent coroutine (or "light th
 
 This API was first enabled in the <code>v0.9.9</code> release.
 
+== ngx.semaphore.new ==
```

Comment:
I think we'd better put the documentation of the `ngx.semaphore` module to the lua-resty-core library instead since the Lua API is defined there. In addition, it deserves a separate documentation file as `lib/ngx/semaphore.md`, just like the module documentation files in the `lua-resty-limit-traffic` library.